contracts/market/src/main.sw

```diff
@@ -551,6 +551,18 @@ impl Market for Contract {
         // Update and write principal to storage
         update_base_principal(caller, user_basic, user_principal_new);
 
+        if user_balance < I256::zero() {
+            // Check that the borrow amount is greater than the minimum allowed
+            require(
+                u256::try_from(user_balance.wrapping_neg())
```

Comment:
Do we want to check the present value or the principal ?

Reply:
I think there is not a lot of difference with such small values, but I think it's fine to check the present value. 

Do you see any problem with that?